### ISSUE TITLE
coverage reporting

### ISSUE THREAD

**Issue by Autoplectic** [dag/attest#77]
attest should have some way to report test coverage, probably using coverage.py.  this is generally a very useful metric and so for attest to gain some real traction needs to support it.

**Comment by dag** [comment]
Doesn't it though work to simply do `coverage run runtests.py`?

If so I still should probably document it.

**Comment by dag** [comment]
With the new `attest` command it isn't obvious how to measure coverage. These should work:

```
coverage run -mattest
coverage run $(which attest)
```

These solutions should be documented as well.

**Comment by dqminh** [comment]
Running `coverage run -mattest` gives me this error: `no such option -m`. My `attest` version is 0.5.2.

**Comment by dag** [comment]
You are quite right, I noticed this too. The coverage docs claim -m is supposed to work but I get the same error. It does however work fine if you point it to a scriptfile that runs the tests.

**Comment by dag** [comment]
[This issue is now resolved upstream in the latest coverage.py relase](https://bitbucket.org/ned/coveragepy/issue/95/run-subcommand-should-take-a-module-name).
